GitForWindowsHelper/github-api-request-as-app.js

```diff
@@ -9,8 +9,8 @@ module.exports = async (context, requestMethod, requestPath, body) => {
     const payload = {
         // issued at time, 60 seconds in the past to allow for clock drift
         iat: now - 60,
-        // JWT expiration time (10 minute maximum)
-        exp: now + (10 * 60),
+        // JWT expiration time (10 minute maximum, use 9 to allow for clock drift)
+        exp: now + (9 * 60),
```

Comment:
I think the 10 minute maximum is supposed to be 10 minutes from `iat`, so with that being offset 60 seconds into the past, `exp` being 9 minutes in the future would make sense.

Reply:
I had the same thought, but why did it seem to work so far, then? It's a puzzle.